README.md

```diff
@@ -156,20 +159,23 @@ named `<accession>-counts.tsv` where each sample is a column and there is a "Gen
 then executing:
 
 ```
-merge_data.py -d data -s "-counts.tsv" -o merged_result.tsv -c condensed_SDRF.tsv -i "Gene ID" --remove-rows-with-empty
+merge_data.py -d data -s "-counts.tsv" -u True -o merged_result.tsv -c condensed_SDRF.tsv -i "Gene ID" --remove-rows-with-empty
```

Comment:
If this is a boolean flag, then just giving the flag should do, please look at argparse documentation for that. You shouldn't need to give True as an argument to the flag.